features.md

```diff
@@ -78,6 +78,35 @@ Access to the email sent by the *Tanker server* is sufficient to retrieve the em
 
 Only the *user* must be able to authenticate to the *application server*, retrieve their identity, and use it to decrypt their *verification key*.
 
+## Passphrase verification method
+
+### Functional perimeter
+
+A *user* can register a passphrase verification method to protect their *verification key*. The passphrase can be a user-chosen password, a randomly-generated token or even the answer to a security question. It *should not* the same as the *application* password.
+
+When *users* needs to verify their identity, they will be asked to enter their passphrase.
```

Comment:
their identities

Reply:
or When **a *user*** needs to verify their identity, they will be asked to enter their passphrase.